src/time_machine/__init__.py

```diff
@@ -121,6 +127,12 @@ def extract_timestamp_tzname(
     elif isinstance(dest, dt.datetime):
         if HAVE_ZONEINFO and isinstance(dest.tzinfo, ZoneInfo):
             tzname = dest.tzinfo.key
+        elif HAVE_PYTZ and isinstance(dest.tzinfo, pytz.BaseTzInfo):
+            raise RuntimeError(
+                "We don't support pytz. For more background information, please read:"
+                " https://blog.ganssle.io/articles/2018/03/pytz-fastest-footgun.html"
```

Comment:
I'm not sure about including a blog post link in an exception message. Put a note and the link in the docs.

Reply:
Ok, Added it there.